pkg/network/encoding/format.go

```diff
@@ -26,6 +26,24 @@ var connPool = sync.Pool{
 	},
 }
 
+var modelAddressPool = sync.Pool{
```

Comment:
I think instead of using pools for `ipTranslation` and the addresses, you can reuse the objects already allocated in the connection objects. e.g.:
```go
func formatAddr(modelAddr *model.Addr, addr util.Address, port uint16, containerID string, ipc ipCache) *model.Addr {
	if addr.IsZero() {
		return nil
	}

    if modelAddr != nil {
    // reuse addr
    }

	return &model.Addr{Ip: ipc.Get(addr), Port: int32(port), ContainerId: containerID}
```

Reply:
Hm... now I'm wondering whether not calling `Reset` on the connection object, and re-using objects like this would produce the same result, since the connection object itself is pooled.

Reply:
I attempted to remove the `c.Reset()` function, then reverted the change and utilized the connection object again. I did not observe any improvements in fact, it appears that the number of allocations increased (944507 allocs/op), when compared to the count following the initial change.